src/explore-education-statistics-common/src/modules/table-tool/components/PublicationForm.tsx

```diff
@@ -93,35 +95,41 @@ const PublicationForm = ({
     </WizardStepHeading>
   );
 
+  const validationSchema = useMemo<ObjectSchema<PublicationFormValues>>(() => {
```

Comment:
Similar to `LocationFiltersForm`, we're exporting the form values type unnecessarily and we could also rename it to `FormValues` for simplicity.